tests/graph/test_all_simple_paths.py

```diff
@@ -176,3 +176,110 @@ def test_digraph_graph_all_simple_paths(self):
         dag.add_node(0)
         dag.add_node(1)
         self.assertRaises(TypeError, retworkx.graph_all_simple_paths, (dag, 0, 1))
+
+
+class TestGraphAllSimplePathsAllPairs(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.edges = [
+            (0, 1),
+            (0, 2),
+            (0, 3),
+            (1, 2),
+            (1, 3),
+            (2, 3),
+            (2, 4),
+            (3, 2),
+            (3, 4),
+            (4, 2),
+            (4, 5),
+            (5, 2),
+            (5, 3),
+        ]
+
+    def test_all_simple_paths(self):
+        dag = retworkx.PyGraph()
+        for i in range(6):
+            dag.add_node(i)
+        dag.add_edges_from_no_data(self.edges)
```

Comment:
Rename dag -> graph, add all nodes in one line.
It's probably better to move these lines in `setUp` since almost all test cases use the same graph.

```suggestion
        graph = retworkx.PyGraph()
        graph.add_nodes_from(range(6))
        graph.add_edges_from_no_data(self.edges)
```

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/577/commits/0647e15b027c036aa32be382689c5de948af4f61